ReleaseHistory.md

```diff
@@ -14,6 +14,7 @@
 * BRK: Remove unused `quiet` parameter from `SarifLogger`. [#2639]https://github.com/microsoft/sarif-sdk/pull/2639
 * BRK: Remove `ComputeHashData` and `AnalysisTargetToHashDataMap` properties from `SarifLogger` (in preference of new `fileRegionsCache` parameter. [#2639](https://github.com/microsoft/sarif-sdk/pull/2639)
 * BRK: Eliminate proactive hashing of artifacts in `SarifLogger` constructor when `OptionallyEmittedData.Hashes` is specified. [#2639](https://github.com/microsoft/sarif-sdk/pull/2639)
+* BUG: Fixed `ERR999.UnhandledEngineException`: `System.InvalidOperationException`: "This operation is not supported for a relative URI" when running in Linux with files skipped due to zero byte size. [#2664](https://github.com/microsoft/sarif-sdk/pull/2664)
```

Comment:
>BUG [](http://example.com/codeflow?start=2&length=3)

please view the PR description in GitHub so it has the screenshots.

Reply:
Put the whole thing from ERR999 to relative URI inside backticks,